x-pack/plugins/maps/public/connected_components/add_layer_panel/view.tsx

```diff
@@ -80,6 +83,7 @@ export class AddLayerPanel extends Component<Props, State> {
       layerWizard,
       layerSteps,
       currentStep: layerSteps[0],
+      editModeEnabled: !!layerWizard.showFeatureEditTools,
```

Comment:
It might be cleaner to not save editModeEnabled to state since layerWizard is already in state. Then just change `if (this.state.layerWizard.showFeatureEditTools) {`

Reply:
Agreed. That's much cleaner